tests/ui/default_numeric_fallback_f64.rs

```diff
@@ -0,0 +1,174 @@
+// aux-build:macro_rules.rs
```

Comment:
Is there a reason, why neither the floating point, nor the integer test are tested with `// run-rustfix`? Looking at the `stderr` files this should be possible.

Could you please test if adding `// run-rustfix` Just Works(tm)?

Reply:
I just forgot to add them.